### DISCUSSION TITLE
Installing On Ubuntu

### DISCUSSION THREAD

**Discussion by parisosuch** [zen-browser/desktop#633] [Q&A] [answered]
Anyone know a comprehensive way to install Zen on Ubuntu? I installed the `.AppImage` and the portable zip. Both of which are lost to me on how to install this permanently to the Ubuntu Apps directory. When I execute the bin `zen` from portable or the `.AppImage`, there isn't even a thumbnail for the app?

**Comment by Samueru-sama** [comment]
>Both of which are lost to me on how to install this permanently to the Ubuntu Apps directory

What do you mean by ubuntus apps directory?

Anyway, try with [this](https://github.com/ivan-hc/AM) and do `am -i zen-browser`

**Reply by parisosuch** [reply]
Like how when I install another app on Ubuntu through the Ubuntu App Store, I am able to search that app and it appears. I was able to figure out how to do it through Flatpack and Flathub!

**Reply by Samueru-sama** [reply]
@parisosuch-dev I think what you were looking for is a snap, which there is an open PR for. 

The flatpak works, though it seems it has some issues that get solved by switching to the appimage, and also I don't think you get architecture specific optimizations with the flatpak.

**Reply by parisosuch** [reply]
Yeah exactly. Apologies, I am not well versed in installing applications on Ubuntu/Linux regularly. I will have to wait for that PR then. For now, flatpack worked and added the app to my searchable apps.

**Reply by Samueru-sama** [reply, marked as answer]
Yeah, the searchable apps are the `.desktop` files in `/usr/share/applications` or `usr/local/share/applications` (for all users) and `$XDG_DATA_HOME/applications` for your user, XDG_DATA_HOME is `$HOME/.local/share` when not set, which is the case 99% of the time. 

For appimages to show in your searchable apps something has to place their desktop entry (they have it inside) in `$XDG_DATA_HOME/applications`, for this you need an appimage manager which there are several including the one I mentioned before, or an [appimage daemon](https://github.com/probonopd/go-appimage/blob/master/src/appimaged/README.md) that actively adds them.

For the portable zip, those are usually extracted in `/opt` instead and you have to make the `.desktop` file on your own.

**Reply by parisosuch** [reply]
I spent the last 20 minutes before this discussion post trying to find the answer to all of those. I was not successful in finding any brief explanation. Thank you so much!
